proposals/lip-0053.md

```diff
@@ -569,7 +571,8 @@ def execute(ccu: Transaction) -> None:
         updateCertificate(ccu)
 
     # Update partner chain outbox root.
-    if ccu.params.inboxUpdate is not empty: # An empty object has all properties set to their default values.
+    # If no certificate is present, the partner chain outbox root would not change.
```

Comment:
I think same modification should be done in line 716 for the sidechain CCU as well, right? 

Also, I would keep the old comment about empty objects.